routes/matches/matches.js

```diff
@@ -775,15 +815,15 @@ router.get("/limit/:limiter", async (req, res, next) => {
     let sql;
     if (req.user !== undefined && Utils.superAdminCheck(req.user)) {
       sql =
-        "SELECT * FROM `match` WHERE cancelled = 0 OR cancelled IS NULL ORDER BY id DESC LIMIT ?,?";
+        "SELECT mtch.* FROM `match` mtch WHERE mtch.cancelled = 0 OR mtch.cancelled IS NULL ORDER BY mtch.id DESC LIMIT ?,?";
     } else {
       sql =
-        "SELECT id, user_id, server_id, team1_id, team2_id, winner, " +
-        "team1_score, team2_score, team1_series_score, team2_series_score, " +
-        "team1_string, team2_string, cancelled, forfeit, start_time, end_time, " +
-        "max_maps, title, skip_veto, private_match, enforce_teams, min_player_ready, " +
-        "season_id, is_pug, map_sides FROM `match` WHERE cancelled = 0 " +
-        "OR cancelled IS NULL ORDER BY id DESC LIMIT ?,?";
+        "SELECT mtch.id, mtch.user_id, mtch.server_id, mtch.team1_id, mtch.team2_id, mtch.winner, " +
```

Comment:
Does this really need to be changed if you haven't made any changes to it aside from adding on the identifier when it's not needed?